praw/reddit.py

```diff
@@ -83,6 +84,23 @@ def read_only(self, value: bool) -> None:
         else:
             self._core = self._authorized_core
 
+    @property
+    def validate_on_submit(self) -> bool:
+        """Get validate_on_submit."""
+        val = self._validate_on_submit
+        if val is False:
+            warn(
+                "Reddit will check for validation on all posts around "
+                "May-June 2020. It is recommended to check for validation.",
```

Comment:
"by setting `reddit.validate_on_submit = True`.

Reply:
That would be a good thing to add.